homeassistant/components/sma/sensor.py

```diff
@@ -45,6 +41,7 @@ async def async_setup_entry(
             SMAsensor(
                 coordinator,
                 config_entry.unique_id,
+                SENSOR_ENTITIES.get(sensor.name),
```

Comment:
Is `sensor.name` of the form `inverter_power_limit` and not `Inverter Power Limit`?

Reply:
Yes, the `pysma` library only uses snake case notation as names.